include/stdexec/__detail/__on.hpp

```diff
@@ -171,9 +171,9 @@ namespace stdexec {
             } else {
               auto&& [__sched, __clsur] = static_cast<_Data&&>(__data);
               return __write_env(                                                       //
-                continue_on(                                                            //
+                continues_on(                                                            //
                   __forward_like<_Data>(__clsur)(                                       //
-                    continue_on(                                                        //
+                    continues_on(                                                        //
```

Comment:
pls run clang-format over this pr.

Reply:
Done. Which version of clang-format do you expect? With 18 I get formatting changes on unrelated parts of the codebase (which I've left out of the changes here), so I was wondering if you expect a specific version. clang-format tends to change its mind about formatting between versions...

Reply:
stdexec has been formatted with clang-format-17. i intend to reformat everything with clang-format-18 as a separate change.